components/x-gift-article/src/Url.jsx

```diff
@@ -22,6 +22,7 @@ export default ({ shareType, isGiftUrlCreated, url, urlType }) => {
 				className={ urlClassNames }
 				disabled={ shareType === ShareType.gift && !isGiftUrlCreated }
 				readOnly
+				aria-label="gift-article-link"
```

Comment:
Why is this kebab case? Wouldn't be better something like...
```suggestion
				aria-label="Gift article shareable link"
```
?
Because that is what is contained in the input.

Reply:
True, not sure what I was thinking!

Reply:
Agree that we should try to give it a name that we would describe it in plain english.

For example: "Shareable article link"

Reply:
Updated as per @umbobabo suggestion. Interestingly VO on mac can read 'gift-article-link' correctly

Reply:
Didn't occur to me either (whoops)!

Interesting to know that VoiceOver can handle kebab-cased text. I know that with uppercase (e.g. HELLO) it will read each letter individually (H-E-L-L-O).